samples/04.0-file-transfer/integration-tests/src/testFixtures/java/org/eclipse/dataspaceconnector/extension/sample/test/FileTransferSampleTestCommon.java

```diff
@@ -44,11 +44,11 @@ public class FileTransferSampleTestCommon {
     //region constant test settings
     static final String INITIATE_CONTRACT_NEGOTIATION_URI = "http://localhost:9192/api/v1/data/contractnegotiations";
     static final String LOOK_UP_CONTRACT_AGREEMENT_URI = "http://localhost:9192/api/v1/data/contractnegotiations/{id}";
-    static final String INITIATE_TRANSFER_PROCESS_URI = "http://localhost:9192/api/v1/data/transferprocess";
+    public static final String TRANSFER_PROCESS_URI = "http://localhost:9192/api/v1/data/transferprocess";
```

Comment:
In reviewing the accessor modifier change, this file (in the this PR branch) indiciates that it `Encapsulates common settings, test steps, and helper methods for the test for samples 4.0 and 4.1.`

should that be updated or more generalized? 

nit: 
my perspective on `common` classes that expose members consumed in other libs should be included in a constants class that can  be referenced in the classes where it is needed.  As of now this class is exposing members from sample 4.0 ref'd 4.1 and 4.2.  Would it be a good practice to create a constants class?  Could be that because these are all 4.X samples it is ok, but something to think about.

Reply:
With @marcgs suggestion there is no need for public constants anymore.

However, thanks for the catch!